CMSIS/Core/Include/cmsis_armcc.h

```diff
@@ -62,9 +62,9 @@
 #ifndef   __STATIC_INLINE
   #define __STATIC_INLINE                        static __inline
 #endif
-#ifndef   __STATIC_FORCEINLINE                 
+#ifndef   __STATIC_FORCEINLINE
```

Comment:
Mixing white space clean up with function enhancements makes it hard to see the relevant ones. Perhaps we spit this into two PRs? I can easily accept the cleanup if it doesn't contain function changes.